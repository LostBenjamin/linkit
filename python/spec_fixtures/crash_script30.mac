

[duplicate-macro-number 10]
    red

[duplicate-macro-number2 10]
    green

