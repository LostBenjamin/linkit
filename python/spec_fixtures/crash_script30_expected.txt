Fixed macro number 10 for macro 'duplicate-macro-number2' is already in use.

